tests/console/yast2_http.pm

```diff
@@ -31,6 +31,9 @@ sub run {
         send_key $cmd{continue};
     }
 
+    # Make sure we do not hit 'alt-i' too early
+    assert_screen 'http_server_deps_install';
```

Comment:
To answer the still open questions regarding if/where we should move this. To my understanding this screen can only appear if in L31 we pressed "continue" to install dependencies so the `assert_screen` should be moved within the if. The verification run also shows that the previous needle matched is "yast2-lan-warning-network-manager". In all other cases we should be on the stable condition of the needle "http-server"

Reply:
No, I do not think so. The `continue` is for the warning for networkmanager, not for the deps.

Reply:
Yes, of course. To my understanding the needle "http-server" should ensure a stable state in the case we do not need to handle the "network manager" warning so IMHO we should move that assert_screen within the if. This also makes sense because otherwise we have two consecutive key presses without synchronisation in between

Reply:
I do not see these 2 warning pop-ups to be related to each other. One is pointing out that the apache2 configuration might not work properly with NM and the other one is installing missing apache2 rpms. AFAIK apache2 and apache2-prefork are standard apache packages. 

```
Information for package apache2:
--------------------------------
Repository     : Main Repository (OSS)                     
Name           : apache2                                   
Version        : 2.4.39-2.2                                
Arch           : x86_64                                    
Vendor         : openSUSE                                  
Installed Size : 4.6 MiB                                   
Installed      : Yes                                       
Status         : out-of-date (version 2.4.39-1.1 installed)
Source package : apache2-2.4.39-2.2.src                    
Summary        : The Apache Web Server Version 2.4         
Description    :                                           
    This version of httpd is a major release of the 2.4 stable branch,
    and represents the best available version of Apache HTTP Server.
    New features include Loadable MPMs, major improvements to OCSP support,
    mod_lua, Dynamic Reverse Proxy configuration, Improved Authentication/
    Authorization, FastCGI Proxy, New Expression Parser, and a Small Object
    Caching API.

     See /usr/share/doc/packages/apache2/, http://httpd.apache.org/, and
    http://httpd.apache.org/docs-2.4/upgrading.html.

Information for package apache2-prefork:
----------------------------------------
Repository     : Main Repository (OSS)                           
Name           : apache2-prefork                                 
Version        : 2.4.39-2.2                                      
Arch           : x86_64                                          
Vendor         : openSUSE                                        
Installed Size : 645.1 KiB                                       
Installed      : Yes                                             
Status         : out-of-date (version 2.4.39-1.1 installed)      
Source package : apache2-2.4.39-2.2.src                          
Summary        : Apache 2 "prefork" MPM (Multi-Processing Module)
Description    :                                                 
    "prefork" MPM (Multi-Processing Module)

    This MPM is basically the one that Apache 1.3.x used. It warrants the
    maximum stability because each server runs in its own process. If a
    process dies it will not affect other servers.
```

Reply:
Maybe we can remove the empty line between assert_screen and send_key calls since these two lines are related to each other.

Reply:
Oh, yes. The needle "http-server" is in fact the same one I am trying to add. So, I can just use an `asser_screen('http-server')` inside the if.
I will push a fixed version. Thanks for the review. :)